packages/sirius-web/frontend/sirius-web-application/src/views/edit-project/workbench-views/DetailsView.tsx

```diff
@@ -63,5 +64,14 @@ export const DetailsView = ({ editingContextId, readOnly }: WorkbenchViewCompone
       </div>
     );
   }
-  return <FormBasedView editingContextId={editingContextId} form={form} readOnly={readOnly} />;
+  return (
+    <div data-representation-kind="form-details">
+      <DetailsContext.Provider
+        value={{
+          detailsPayload: payload,
+        }}>
+        <FormBasedView editingContextId={editingContextId} form={form} readOnly={readOnly} />
+      </DetailsContext.Provider>
```

Comment:
Why not reuse the same FormContext.Provider here?
It seems like to use the payload a component inside a FormBasedView would need to handle both kinds of contexts, and to be aware of the detailsPayload it would need to depend on sirius-web-application.